BotController/BotController.cpp

```diff
@@ -1,12 +1,340 @@
-// BotController.cpp : This file contains the 'main' function. Program execution begins and ends there.
-//
-
 #include <iostream>
+#include <Winuser.h>
 #include "Robot.hpp"
+//include "RGBQuadValues.hpp"
 
 int main()
 {
-    std::cout << "Hello World!\n";
+    Robot robot = Robot();
+
+    //how exact the pixel color's RGBs need to be
+    double tolerance = 0.0;
+
+    //in milliseconds
+    DWORD clickSpeed = 80;
```

Comment:
I could also have it so that instead of setting the delay in the parameters of each function call, it could be an accessible member of `Robot` and therefore you would set the delay and it is fixed until it is reset. This behaves more like Java's bot I think.